### PR TITLE
Documentation updates for v0.9.0-rc2

### LINKED ISSUE #92: release process docs need cleaned up
release.md has old references to the gitlab account that should be update to reflect the new github account.

### LINKED ISSUE #107: tasks missing in intro docs
There should be  a tasks section here: https://quintilesims.github.io/layer0/intro/

### PR DESCRIPTION
**Pull request by tlake**
- Added brief description of tasks to intro (fixes #107)
- Added defaults for healthchecks in loadbalancer create
- Updated release process notes for github (fixes #92)

### CODE REVIEW

File: docs-src/docs/intro.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ SSL certificates obtained from a valid [Certificate Authority (CA)](https://en.w
 
 [ECS Task Definitions](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_defintions.html). These configuration files detail how to deploy your application. We have several [sample applications](https://github.com/quintilesims/layer0-examples) available that show what these files look like --- they're called `Dockerrun.aws.json` within each sample app.
 
+### Tasks
+
+One-off commands that don't necessarily make sense to keep running, or to restart when they finish.
```
Comment (tlake) [inline docs-src/docs/intro.md]: Should be a bit better, I think.

File: RELEASE.md [hunk 2]
```diff
@@ -79,13 +96,17 @@ make release
...
 
+
 # Announce the release
+
 Once the release is available for download, send a message to
 [#xfra](https://ims-dev.slack.com/messages/xfra) with a link and a one-line
 summary of the release contents. For example:
 ```
-@here Layer0 has released v0.5.2 http://docs.xfra.ims.io/releases - which aims to be a stable version of our load balancer support.
+@here Layer0 has released v0.9.0 http://docs.xfra.ims.io/releases - which aims to be a stable version of our load balancer support.
```
Comment (zpatrick) [inline RELEASE.md]: I think we are going to depreciate docs.xfra.ims.io, this should probably be the github pages url

Comment (tlake) [inline RELEASE.md]: Oh crap, yeah, good catch! I meant to fix that while I was right there - guess I got distracted xD

✅

**Comment by zpatrick** [comment]
:+1:

**Pull request merged**